src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -263,7 +281,7 @@ const DomainFragment = ({
             item={{
               name: item.domain_name,
               image: item.logo,
-              isunfollowed: !item.following,
+              isunfollowed: ifollow.length === 0 ? !item.following : !isFollow,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for determining whether a domain is unfollowed has been improved. This is a good change as it takes into account the current state of followed domains.
```diff
+ isunfollowed: ifollow.length === 0 ? !item.following : !isFollow,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->